src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -230,8 +230,14 @@ private async Task GenerateDeploymentRecipeSnapShot(Recommendation recommendatio
             recipe.Name = recipeName;
             recipe.CdkProjectTemplateId = null;
             recipe.CdkProjectTemplate = null;
+            recipe.PersistedDeploymentProject = true;
 
-            var recipeSnapshotBody = JsonConvert.SerializeObject(recipe, Formatting.Indented);
+            var recipeSnapshotBody = JsonConvert.SerializeObject(recipe, new JsonSerializerSettings
```

Comment:
Enum are being serialized as int. For example below you can see type and type hint are saved as ints.

```
      "Type": 3,
      "TypeHint": 8,
```